docs/pages/server-access/architecture.mdx

```diff
@@ -0,0 +1,228 @@
+```
+
+Note the "Labels" column on the farthest side.
+
+### Labels
+
+Labels are permanent or temporary taxonomic handles for grouping resources. They can be used to group resources temporarily, across multiple clusters, or to define a grouping useful for querying without modifying the underlying access permissions or cluster membership:
+
+- `tsh ssh root@env=example` returns the list of all resources with the `env=example` label specified in their `teleport.yaml` (`'env': 'example'`).
+- Suppose an IP address changes, an admin can quickly find the current Node with that label since it remains unchanged.
+- Suppose you need to query the SSH Node of each cluster. Give each SSH Node the same label so a single query will allow you to introspect those specific resources across all of your clusters.
+
+This was designed in tandem with the Unified Resource Catalog to make it easier to manage mass-scale deployments (common to many industries like IoT and manufacturing).
+
+### Multi-querying
+
+`tsh` supports [multi-querying](../cli-docs.mdx#tsh): the ability to execute sequential (or simultaneous) commands on multiple resources at once. 
+
+Multi-querying allows the same command to be executed on multiple machines but typed once. This is convenient for scenarios where an operation is a short command executed routinely (`apt-get update`). Admins need not access each machine individually, key-by-key. 
+
+Instead, all resources within a query will execute the supplied command.
+
+For example:
+
+```bash
+# Execute ls on each resource with the label env=example
+tsh ssh root@env=example ls
+```
+
+will execute the `ls` command on each Node with the `env=example` label, displaying the contents of each to the screen.
+
+## Defining access request workflows
+
+<Admonition title="tip" type="tip">
+  Access Requests and Access Request workflows are now available in both the Teleport Open Source and Enterprise Editions.
+</Admonition>
+
+Granting and revoking permissions is a natural part of any system for managing identities, information, and access to resources.
+
+Teleport provides built-in support for [requesting, renewing, granting, revoking, and denying permissions](../enterprise/workflow/index.mdx) through the same Web Interface and CLI used to access permitted resources.
+
+## Auditing and events
+
+Teleport provides powerful [Session Recording](../features/enhanced-session-recording.mdx) and [Audit Logging](../admin-guide.mdx#audit-log) that can be added to your existing SSH solution:
+
+- User sessions are recorded and can be replayed. 
+- They can also be shared, stored, and organized by query and triggered Events.
+
+### Defining Events
+
+Teleport admins can further define [Events](../admin-guide.mdx#events) to organize and trigger sending session recordings to customizable storage back-ends:
+
+1. All `session.start` events could be stored on AWS S3.
```

Comment:
We support a range of backends, S3, S3 Compatible and GCP File store.  I would make this section less vendor specific